api/src/types.rs

```diff
@@ -266,7 +266,7 @@ pub struct OutputPrintable {
 	/// Rangeproof hash (as hex string)
 	pub proof_hash: String,
 	/// Block height at which the output is found
-	pub block_height: Option<u64>,
```

Comment:
Why the change here? Just wondering if anything in the wallet will be expecting a ‘null’ here

Reply:
I don't remember why I changed the structure from `Option<u64>`to `u64`. Reverting.